react-first-app/src/redux/tictactoe/reducers/index.js

```diff
@@ -0,0 +1,10 @@
+import { combineReducers } from 'redux'
+import tictactoe from './tictactoe'
+import { reducer as formReducer } from 'redux-form';
+import routerReducer from 'react-router-redux';
+
+export default combineReducers({
+  tictactoe,
```

Comment:
Be consistant in naming stuff. Use `fooReducer` or `foo` for all reducers, choose one.